src/services/walktroughServices.js

```diff
@@ -0,0 +1,109 @@
+import { create, watch, currentUser, OpenShiftWatchEvents } from './openshiftServices';
+import {
+  cleanUsername,
+  findOpenshiftResource,
+  buildServiceInstanceResourceObj,
+  findOrCreateOpenshiftResource
+} from './middlewareServices';
+import * as osKinds from '../common/openshiftKinds';
+import { middlewareTypes } from '../redux/constants';
+import { FULFILLED_ACTION } from '../redux/helpers';
+import { DefaultServiceInstanceTransform } from '../common/serviceInstanceHelpers';
+
+/**
+ * Node messaging app transform, adds enmass credentials
+ */
+class NodeMessagingAppEnmassSiTransform {
+  isTransformable(siInfo) {
+    return siInfo.name === 'nodejs-messaging-work-queue-frontend';
+  }
+
+  transform(siInfo) {
+    const enmassCreds = {
+      MESSAGING_SERVICE_USER: 'username',
+      MESSAGING_SERVICE_PASSWORD: 'password'
+    };
```

Comment:
We'll probably need to set the internal service address here too to override the default URL.

Reply:
@aidenkeating Is that this URL here? `amqp://messaging.enmasse-eval.svc:5672?amqp.saslMechanisms=PLAIN`